packages/components/src/PieChart/PieChart.component.js

```diff
@@ -18,7 +18,9 @@ export function decorateWithTooltip(tooltip, label, tooltipPlacement, Component,
 	}
 	return (
 		<TooltipTrigger label={label} tooltipPlacement={tooltipPlacement}>
-			<Component {...props} />
+			<span>
```

Comment:
Why this span ?

Reply:
Tooltip trigger can't place a ref on a functional component
```
Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

Check the render method of `TooltipTrigger`.
    in withI18nextTranslation(PieChartIcon) (created by TooltipTrigger)
    in TooltipTrigger (created by PieChart)
    in PieChart
```